prefpicker/templates/browser-fuzzing.yml

```diff
@@ -359,6 +359,11 @@ pref:
     - null
     fission:
     - true
+  fission.sessionHistoryInParent:
+    default:
+    - false
```

Comment:
Should `default` entry be `null` to use the browser default instead?

Reply:
Yes.  I didn't realize that's what null represented.